ArchUnitNETTests/Domain/MemberTests.cs

```diff
@@ -279,16 +325,22 @@ public class FieldType
         
         private class ClassReadOnly
         {
-            private readonly string readonlyVar;
-            private readonly string readonlyVarInit = "";
-            private string NotReadOnlyVarInit = "";
-            private string PropertyWithoutSet { get; }
-            private string InitOnlyProperty { get; init; }
-            private string PropertyWithGetAndSet { get; set; }
+            public readonly string ReadonlyVar;
+            public readonly string ReadonlyVarInit = "";
+            public string NotReadOnlyVarInit = "";
+            public string PropertyWithoutSet { get; }
+            public string InitOnlyProperty { get; init; }
+            public string PropertyWithGetAndSet { get; set; }
```

Comment:
Is the case of privately accessible properties still covered in tests?

Reply:
For those tests, it doesn't matter if those properties are public, just like it didn't matter that they were private before. I changed it to be able to use `nameof` instead of hardcoding their names in tests.